webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/snippet/node/Groups.scala

```diff
@@ -159,7 +160,7 @@ class Groups extends StatefulSnippet with DefaultExtendableSnippet[Groups] with
               // no modification, so no refreshGroupLib
               refreshTree(htmlTreeNodeId(groupId)) &
               showGroupSection(Right(fullGroupTarget.nodeGroup), lib.categoryByGroupId(gid)) &
-              JsRaw("createTooltip()")
+              JsRaw("createTooltip()") // JsRaw ok, const
```

Comment:
There is a way that the `refreshTree` could be subject to injection here : in its implementation, there are still unescaped `JsRaw`s in the `buildGroupTree` method